karma.conf.js

```diff
@@ -15,8 +15,8 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
-      './js/main/*.js',
-      './js/test/*.js'
+        './public/js/*.js',
```

Comment:
Following the style guide on the [wiki](https://github.com/DanyDin1801/Product-Portal/wiki), javascript files should be indented with two spaces.